### ISSUE TITLE
xpu: set of not implemented aten ops affecting huggingface tests

### ISSUE THREAD

**Issue by dvrogozh** [pytorch/pytorch#128914]
With the xpu support landed in huggingface (use https://github.com/huggingface/accelerate/commit/3b5a00e048f4393398d8ea8c4f468857f595f039 and https://github.com/huggingface/transformers/commit/eed9ed679878ada2f6d2eefccdbda368cabc88b1 or later), there are few aten ops not implemented in xpu which affect common huggingface tests. Please, implement:

* `tests/trainer/test_trainer.py`:
  - [x] `aten::addcdiv.out`, https://github.com/intel/torch-xpu-ops/pull/486
  - [x] `aten::addcmul.out`, https://github.com/intel/torch-xpu-ops/pull/418
  - [x] `aten::equal`, https://github.com/intel/torch-xpu-ops/pull/550
  - [x] `aten::lerp.Scalar_out`, https://github.com/intel/torch-xpu-ops/pull/449
  - [x] `aten::linalg_vector_norm.out`, https://github.com/intel/torch-xpu-ops/pull/460
  - [x] `aten::mse_loss_backward`, https://github.com/intel/torch-xpu-ops/pull/438
  - [x] `aten::mse_loss.out`, https://github.com/intel/torch-xpu-ops/pull/438
  - [x] `aten::norm.out`, https://github.com/intel/torch-xpu-ops/pull/556

* `tests/test_modeling_utils.py`:
  - [x] `aten::logical_or.out`, https://github.com/intel/torch-xpu-ops/pull/529

* `tests/pipelines/*`:
  - [x] `aten::isin.Tensor_Tensor_out`, https://github.com/intel/torch-xpu-ops/pull/548
  - [x] `aten::multinomial`, https://github.com/intel/torch-xpu-ops/pull/520
  - [x] `aten::topk.values`, https://github.com/intel/torch-xpu-ops/pull/547

CC: @gujinghui @EikanWang @fengyuan14 @guangyey @jgong5

cc @gujinghui @EikanWang @fengyuan14 @guangyey

**Comment by fengyuan14** [comment]
Except for `equal` and `isin`, others are included in PT2.5 support plan. We will evaluate the efforts to see whether to add them.

**Comment by dvrogozh** [comment]
First group of ops landed after https://github.com/pytorch/pytorch/pull/129353 got merged. See updated issue description.

**Comment by fengyuan14** [comment]
> Except for `equal` and `isin`, others are included in PT2.5 support plan. We will evaluate the efforts to see whether to add them.

@dvrogozh These two operators has been added to PT2.5 support list.

**Comment by dvrogozh** [comment]
@fengyuan14 : fyi, not implemented ops which I reported before after running `tests/pipelines/*` also affect running models from HF model cards. For example, this happens running aten::isin.Tensor_Tensor_out, https://github.com/intel/torch-xpu-ops/pull/548. I believe we already target them PT2.5, let's make sure we don't miss this. Ops are:
- [ ] `aten::isin.Tensor_Tensor_out`, https://github.com/intel/torch-xpu-ops/pull/548
- [ ] `aten::multinomial`, https://github.com/intel/torch-xpu-ops/pull/520
- [ ] `aten::topk.values`, https://github.com/intel/torch-xpu-ops/pull/547

Ran aten::isin.Tensor_Tensor_out, https://github.com/intel/torch-xpu-ops/pull/548 using this script
```
import torch
import transformers
from transformers import AutoTokenizer
from transformers import pipeline

name = 'mosaicml/mpt-7b'

model = transformers.AutoModelForCausalLM.from_pretrained(
  name,
  torch_dtype=torch.bfloat16, # Load model weights in bfloat16
  trust_remote_code=True)

tokenizer = AutoTokenizer.from_pretrained('EleutherAI/gpt-neox-20b')

pipe = pipeline('text-generation', model=model, tokenizer=tokenizer, device='xpu:0')

with torch.autocast('xpu', dtype=torch.bfloat16):
    print(
        pipe('Here is a recipe for vegan banana bread:\n',
            max_new_tokens=100,
            do_sample=True,
            use_cache=True))
```

With:
* https://github.com/pytorch/pytorch/commit/3477ee38e4dd1429ecfd7e6f20a30cce0f4f78e7
* https://github.com/pytorch/vision/commit/8f9d810a26f1e3be97e8ec48a214967accdb9016
* https://github.com/intel/torch-xpu-ops/commit/4db0b0cd1ca51d9cfd890be2eb3527b165782220
* https://github.com/huggingface/accelerate/commit/12a007d55937345aa986f5d7b1a1b6f2038465a7
* https://github.com/huggingface/transformers/commit/3345ae733b6f4aeb7204a0f3e646a3cdbaad0023

**Comment by dvrogozh** [comment]
Few more ops are now available after #131015 merged. Remainder:
- [ ] `aten::norm.out`, https://github.com/intel/torch-xpu-ops/pull/556
- [ ] `aten::topk.values`, https://github.com/intel/torch-xpu-ops/pull/547

**Comment by dvrogozh** [comment]
All tasks done after https://github.com/pytorch/pytorch/pull/131643 merged.
